test/test_security_service.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, methodName, provider):
             methodName=methodName, provider=provider)
 
     def test_crud_key_pair_service(self):
-        name = 'cbtestkeypair-a'
+        name = 'cbtestkeypairA-{0}'.format(uuid.uuid4()).lower()
```

Comment:
Thanks for reviewing. This renaming is not really needed since the test security groups and networks are deleted after running each test. I just wanted to make it more like what we have on the master branch.